src/language-js/print/misc.js

```diff
@@ -57,6 +57,15 @@ function printDeclareToken(path) {
   );
 }
 
+const tsAbstractNodeTypes = new Set([
+  "TSAbstractMethodDefinition",
+  "TSAbstractPropertyDefinition",
+  "TSAbstractAccessorProperty",
+]);
+function printAbstractToken({ node }) {
```

Comment:
Why received params as an object?
```suggestion
function printAbstractToken(node) {
```

Reply:
The paramter is AstPath like other `print*` functions.

Reply:
I didn't notice it is an AST path!